site/src/examples/button/Loading.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from "@salt-ds/core";
+import { useState } from "react";
+
+function fetchPDFDocument() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      return resolve({});
+    }, 2000);
+  });
+}
+
+export function Loading() {
+  const [loading, setLoading] = useState(false);
+  const [loadingAnnouncement, setLoadingAnnouncement] = useState("");
```

Comment:
Does this still need to be state? `loadingAnnouncement` can be set inline on line 29

Reply:
Agree to keep it for now